### DISCUSSION TITLE
Hide systray notifications

### DISCUSSION THREAD

**Discussion by gruvw** [qtile/qtile#3909] [Q&A] [answered]
Hello,

I have the Synology Drive application installed and it is sending me some notifications via the systray.  
![image](https://user-images.githubusercontent.com/63407038/196121647-2a97bbb5-a8c8-401f-bf4c-af266bc1093b.png)
How to disable the display of systray notifications?

**Comment by elParaguayo** [comment, marked as answer]
I think you'd need to disable that in the app. This isn't controlled by qtile.
